onnx/onnx.in.proto

```diff
@@ -456,6 +456,11 @@ message TensorProto {
   repeated uint64 uint64_data = 11 [packed = true];
 }
 
+// TensorProtos stores multiple TensorProto objects in one single proto.
+message TensorProtos {
```

Comment:
Wanna try sequence: https://github.com/onnx/onnx/blob/master/onnx/onnx.proto#L515

Reply:
How to use this?